src/lib/components/consignments/index.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import Button from "../buttons/inverted_button"
+import Welcome from "./setup/welcome"
+
+import {
+  LayoutChangeEvent,
+  NavigatorIOS,
+  ScrollView,
+  Text,
+  TouchableHighlight,
+  View,
+  ViewProperties,
+} from "react-native"
+
+/** The metadata for a consigned work */
+interface ConsignmentMetadata {
+  title: string | null
+  year: string | null
+  category: string | null
+  materials: string | null
+  width: number | null
+  height: number | null
+  depth: number | null
+  units: "in" | "cm"
+}
+
+interface Props extends ViewProperties {
+  artist?: any
+  photos?: string[]
+  metadata?: ConsignmentMetadata
+  location?: string
+  provenance?: string
+}
+
+export default class Consignments extends React.Component<Props, any> {
+  // constructor(props) {
```

Comment:
commented code

I'm assuming you're gonna clean this up in a subsequent PR but in case you forgot